src/storage/localStorage.ts

```diff
@@ -1,6 +1,12 @@
 type StorageObject = { [name: string]: any };
 
+// Creating this as a dict, as we might want to save some metadata under the same key.
```

Comment:
Remove.

Add a last-modified-date field into `CompanyData`? This field could be used to make decisions about whether to update the local storage.